app/storage/redis.py

```diff
@@ -36,7 +40,7 @@ def put(self, model, overwrite=True):
                 name=key_value, value=value, ex=expires_in, nx=not overwrite
             )
         except RedisConnectionError:
-            logger.info("retrying redis set command")
+            logger.info(self.command_failed_log_message, command="set")
```

Comment:
Seeing as the log message has been factored out into a method, the method could do the logging. The log lines would then be something like `log_retry(command="set")`.

Reply:
Done 👍🏻